packages/global-styles-ui/src/variations/variation.tsx

```diff
@@ -41,7 +44,7 @@ export default function Variation( {
 	} = useContext( GlobalStylesContext );
 
 	const context = useMemo( () => {
-		let merged = { ...base, ...variation };
+		let merged = mergeGlobalStyles( base, variation );
```

Comment:
Makes sense. The code was previously using `mergeBaseAndUserConfigs` private util, which was doing the same thing as `mergeGlobalStyles`.